src/AppBundle/Form/MembershipChooseNearbyCommitteeType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Entity\Committee;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class MembershipChooseNearbyCommitteeType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('committees', EntityType::class, [
```

Comment:
Ok nervermind (note that the prefix in this case is defined by `getBlockPrefix()` in the type).